src/screens/Login/Login.jsx

```diff
@@ -145,9 +145,10 @@ const Login = ({ image }) => {
                 value={serverAddressInput}
               />
             )}
-            <ServerValidIndicator setValid={setIsServerValid} />
-            {serverError && (
+            {serverError ? (
```

Comment:
shouldn't we improve this also in Register.jsx?

Reply:
Yes sir.